sorcery.gemspec

```diff
@@ -30,5 +30,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency "simplecov", ">= 0.3.8"
   s.add_development_dependency "rspec", "~> 3.0.0"
   s.add_development_dependency "rspec-rails", "~> 3.0.0"
+  s.add_development_dependency "test-unit", "~> 3.0.0"
```

Comment:
It's just mine local change, because of rspec won't to work. It's shoudnt be in git